kubernetes-model/src/main/resources/schema/kube-schema.json

```diff
@@ -12600,6 +12600,10 @@
       "$ref": "#/definitions/kubernetes_extensions_Deployment",
       "javaType": "io.fabric8.kubernetes.api.model.extensions.Deployment"
     },
+    "DeploymentConfig": {
```

Comment:
Were these lost in a previous generation? These have always been there before, no idea how they're newly added now?

Reply:
In fact me I was surprised for this